engine/immutable/merge_tool_test.go

```diff
@@ -673,3 +674,20 @@ func TestCompactionDiffSchemas(t *testing.T) {
 
 	assert.NoError(t, compareRecords(mh.readExpectRecord(), mh.readMergedRecord()))
 }
+
+func TestItrTSSPFile(t *testing.T) {
+	pathName := "00000001-0000-00000000.tssp"
```

Comment:
It should be better to generate the tssp file here, but not put the file into the repo.